genienlp/arguments.py

```diff
@@ -496,9 +496,7 @@ def parse_argv(parser):
         'ngrams: lookup all ngrams in the text and see if there is a match',
     )
 
-    parser.add_argument(
-        '--almond_domains', nargs='+', default=[], help='Domains used for almond dataset; e.g. music, books, ...'
-    )
+    parser.add_argument('--ned_domains', nargs='+', default=[], help='Domains used for almond dataset; e.g. music, books, ...')
```

Comment:
You still have ned_domains and ner_domains? Is that necessary?

Reply:
`ner_domains` are for CrossNER task since it has multiple domains. I can rename it to `crossner_domains`